### ISSUE TITLE
Add diff mode to iterations.

### ISSUE THREAD

**Issue by alex-bender** [exercism/exercism.io#2576]
It would be great to highlight differences between iterations, coz now it is little bit complicated to
figured out, what have been changed if code is messy.

**Comment by kytrinyx** [comment]
We used to have this (see #833), but the interface got too messy and too confusing, so we removed it. People still ask for it (#1314), but this will have to wait until someone tackles a full redesign of the code page.

**Comment by alex-bender** [comment]
Thank you for reply and sorry for duplicating issues.

**Comment by kytrinyx** [comment]
No, worries! Eventually I'll figure out how to put up a proper roadmap :)

**Comment by GustavoCaso** [comment]
@kytrinyx maybe I can help you figure a proper roadmap to bring this feature to live.

**Comment by kytrinyx** [comment]
@GustavoCaso I'm not sure exactly what you mean. Do you want to discuss this particular feature? Or discuss the roadmap?

**Comment by GustavoCaso** [comment]
@kytrinyx I mean discuss the how we can achieve this feature, work on the issue. Sorry for my English. :-)

**Comment by kytrinyx** [comment]
I'd actually like to hold off on this feature for now. I feel like we need to do proper design of the code page before we can start working on this. The reason I took it out in the first place is because the user interface already has so much stuff in it. This would add even more complexity to what is possibly the most important page in the app.

**Comment by GustavoCaso** [comment]
@kytrinyx Great, Sound logical. So I will go with another issue. :-)

**Comment by petertseng** [comment]
It'd be super rad if this could happen, but I get by in the meantime by using `exercism download` on the two iterations in question and then using `diff` on them locally. So I'm not too high-pri on it I guess

**Comment by kytrinyx** [comment]
Nice workaround, @petertseng.

**Comment by kytrinyx** [comment]
I'm going to close this since there is a workaround. We're working on improving the core features / usability of the site, and once we have that I'm sure this will come up again :)
